test/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const PROJECT_ROLE_REVERT_MSG = "'ProjectRole: caller is not the project wallet'";
+export const OWNER_REVERT_MSG = "'Ownable: caller is not the owner'";
+export const CALLER_NOT_OWNER_REVERT_MSG = "'Ownable: caller is not the owner'";
```

Comment:
nit - what's the difference between these two?

Reply:
Right, this is a duplicate. Will fix in a future PR